app/commands/user/authenticate_from_omniauth.rb

```diff
@@ -18,6 +19,7 @@ def find_by_uid
         user.save
       end
 
+      user.update(github_username: auth.info.nickname)
```

Comment:
So, fun one for you here. This will fail if for some reason the user is not "valid" for some unrelated reason. It'll fail silently. We could use `update!` which would raise an exception, but we don't want that. So instead we can use `update_column(:github_username, auth.info.nickname)` which bypasses all the Rails checks and writes it even if the model is invalid.

Reply:
Interesting!